array-manipulation-1/src/head.ts

```diff
@@ -0,0 +1,5 @@
+/* exported head */
+
+function head(array: any[]): unknown {
```

Comment:
best to keep types consistent: If you use any[] then return type any. If you use unknown[] then return type unknown...